src/transformation/mesh_intersection/triangle_triangle_intersection.rs

```diff
@@ -250,6 +175,40 @@ pub fn triangle_triangle_intersection(
     }
 }
 
+/// Smarter, but more expensive, mechanism to find a triangle normal.
+fn robust_triangle_normal(tri: &Triangle) -> na::Vector3<Real> {
```

Comment:
This should be a method in the `Triangle` struct. I’ve made [a comment](https://github.com/dimforge/parry/pull/221#discussion_r1674206434) in the previous review but that likely got unnoticed.